[ENH] BEP001 - Entity-linked file collections (#668)
bids-standard/bids-specification: src/02-common-principles.md

specified is available in the [entity table](./99-appendices/04-entity-table.md)
in the appendix.

### Entity-linked file collections

A set of files that are related to each other based on a repetitive acquisition of
sequential data by changing acquisition parameters one (or multiple) at a time and/or by
being inherent components of the same data. The suffix serves as an identifier for these
collections, holding a logical grouping of constituent files.
Within each collection, files MUST be distinguished from each other by at least one 
entity (e.g., `echo`) that corresponds to an altered acquisition parameter (e.g., 
`EchoTime`) or that defines a component relationship (e.g., `part`). Note that these entities 
MUST be described by the specification and the parameter changes they declare SHOULD NOT invalidate
the definition of the accompanying suffix. For example, the use of the `echo` entity along with the 
`T1w` suffix casts doubt on the validity of the identified contrast weighting.  

> Provided the conditions above are satisfied, any suffix (e.g. `bold`) can identify an entity-linked file 
> collection, although certain suffixes are exclusive for this purpose (e.g., `MP2RAGE`).

Reviewer: ```suggestion
Provided the conditions above are satisfied, any suffix (such as `bold`) can identify an entity-linked file 
collection, although certain suffixes are exclusive for this purpose (for example, `MP2RAGE`).
```

> All the use cases concerning this convention MUST to be included in the [file collections](./99-appendices/10-file-collections.md) appendix. This convention is mainly intended for but not

Author: Thank you @effigies I was not sure about adding this, I will make this point in the DECISION_MAKING.md document.
